src/encoded/static/vis_defs/WGBS_vis_def.json

```diff
@@ -17,7 +17,7 @@
                     "tag": "aCpG",
                     "visibility": "dense",
                     "type": "bigBed",
-                    "file_format_type": [ "bedMethyl" ],
+                    "file_format_type": ["bedMethyl", "bed 9+"],
```

Comment:
I think it should be "bed9+" rather than "bed 9+"

Reply:
thanks, fixed